src/service/viewer-impl.js

```diff
@@ -320,7 +320,7 @@ export class ViewerImpl {
         let error = getChannelError(
           /** @type {!Error|string|undefined} */ (reason)
         );
-        if (error && error.message === timeoutMessage) {
+        if (error && error.message.endsWith(timeoutMessage)) {
```

Comment:
Use the `endsWith` helper in src/string.js.

https://caniuse.com/#feat=mdn-javascript_builtins_string_endswith

Reply:
Good ol' IE11, I had no idea this wasn't supported :man_facepalming: good catch. Done!